packages/cli/src/UserManagement/routes/passwordReset.ts

```diff
@@ -43,15 +43,17 @@ export function passwordResetNamespace(this: N8nApp): void {
 
 			const user = await Db.collections.User!.findOne({ email });
 
-			if (!user) {
+			if (!user || !user.password) {
 				return;
```

Comment:
should this perhaps throw a "500 - Internal Server Error"?

Reply:
I vote for returning a successful response for password requests always and response should be in the format "An email will arrive shortly if the email xyz exists in our database".

This exposes less details about existing users.

Reply:
ah, okay. Makes sense.